### ISSUE TITLE
RT Sample DB

### ISSUE THREAD

**Issue by DocCyblade** [DocCyblade/tkl-rt#19]
Create sample database to use for RT install

**Comment by DocCyblade** [comment]
Not sure if we want demo data, as we use the RT setup program to setup the database.

**Comment by JedMeister** [comment]
Having sample data can be nice from a new user perspective, especially if you are just testing to see if the software is for you. 

Although the downside of it can be getting a clean start when you decide you do want to go with it.

**Comment by DocCyblade** [comment]
Yeah sample data would be nice. I am thinking of scripting a setup script that can setup the database and an option could be add sample data. I think RT may even have something of a demo project too.
I moved it to beta target as I just want to ge an functional RT build for alpha1.
